test/e2e/autoscale.go

```diff
@@ -201,12 +209,11 @@ func SetupSvc(t *testing.T, class, metric string, target int, targetUtilization
 	clients := Setup(t)
 
 	t.Log("Creating a new Route and Configuration")
-	names := test.ResourceNames{
+	names := &test.ResourceNames{
 		Service: test.ObjectNameForTest(t),
 		Image:   autoscaleTestImageName,
 	}
-	test.EnsureTearDown(t, clients, &names)
```

Comment:
Is this EnsureTearDown is no longer necessary? If so the comment line on this func:
> // It sets up EnsureTearDown to ensure that resources are cleaned up when the
// test terminates.

https://github.com/mgencur/serving-1/blob/migrate_upgrade_tests/test/e2e/autoscale.go#L205-L206
should be updated.

Reply:
Right, I'll remove the comment.

Reply:
Done.